harper-core/dictionary.dict

```diff
@@ -52787,6 +52787,7 @@ shout-out/1S
 sign on/4
 sign-on/1MS
 single-handedly/j
+handedly
```

Comment:
Careful, that's a single-word word in the middle of the multi-word section (-:

Then again, maybe it's time to just fully accept multi-word entries and sort them together with the main part? Leaving just the "specialist" words in their own section?